packages/infrastructure/typescript/config/base.json

```diff
@@ -17,6 +17,7 @@
     "noImplicitAny": true,
     "noUnusedLocals": true,
     "noImplicitReturns": true,
+    "experimentalDecorators": true,
```

Comment:
We don't need this anymore, as we don't plan to extract anything with decorators? Let's keep it around anyway just in case it will save some effort.

Reply:
Also this is not needed in Typescript v5 since it's support decorators out of the box.

@Iku-turso maybe we should plan to do something with them since it's not experimental anymore.

PTAL:
https://2ality.com/2022/10/javascript-decorators.html
https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#decorators

Reply:
Should I remove this then?